packages/protocol/contracts/automata-attestation/AutomataDcapV3Attestation.sol

```diff
@@ -92,7 +100,7 @@ contract AutomataDcapV3Attestation is IAttestation, EssentialContract {
                 continue;
             }
             serialNumIsRevoked[index][serialNumBatch[i]] = true;
-            // TODO(yue): emit an event
+            emit AddRevokedCertSerialNum(index, serialNumBatch[i]);
```

Comment:
how about rename `addRevokedCertSerialNum` to `revokeCertSerialNum`, then rename event to `CertSerialNumRevoked`.

I also suggest to rename other events to xxxAdded, xxxRemoved, xxxConfigured, xxxSet